velox/exec/tests/AggregateFunctionRegistryTest.cpp

```diff
@@ -266,7 +266,8 @@ TEST_F(FunctionRegistryTest, multipleNames) {
       "aggregate_func2", {BIGINT(), DOUBLE()}, BIGINT(), ARRAY(BIGINT()));
 
   auto registrationResults = registerAggregateFunction(
-      {std::string("aggregate_func2"), std::string("aggregate_func3")},
+      std::vector<std::string>{
+          std::string("aggregate_func2"), std::string("aggregate_func3")},
```

Comment:
Do we still need `std::string` for the elements with this change?

Reply:
@majetideepak thanks for the tip! just removed the "std::string()"